cassandane/tiny-tests/Caldav/invite_lots_of_attendees

```diff
@@ -0,0 +1,89 @@
+#!perl
+use Cassandane::Tiny;
+
+sub test_invite_lots_of_attendees
```

Comment:
Can this test deterministically reproduce the issue? If not, maybe we should get rid of it, as it will only make the tests run longer?